clients/metadata/metadata_cassandra.go

```diff
@@ -88,42 +92,56 @@ const (
 	columnBeginEnqueueTime               = "begin_enqueue_time"
 	columnBeginSequence                  = "begin_sequence"
 	columnBeginTime                      = "begin_time"
+	columnCallerHostName                 = "caller_host_name"
+	columnCallerServiceName              = "caller_service_name"
+	columnChecksumOption                 = "checksum_option"
 	columnConnectedStore                 = "connected_store"
 	columnConsumedMessagesRetention      = "consumed_messages_retention"
 	columnConsumerGroup                  = "consumer_group"
 	columnConsumerGroupUUID              = "consumer_group_uuid"
 	columnConsumerGroupVisibility        = "consumer_group_visibility"
 	columnCreatedTime                    = "created_time"
 	columnDLQConsumerGroup               = "dlq_consumer_group"
+	columnDLQMergeBefore                 = "dlq_merge_before"
+	columnDLQPurgeBefore                 = "dlq_purge_before"
 	columnDeadLetterQueueDestinationUUID = "dead_letter_queue_destination_uuid"
 	columnDestination                    = "destination"
 	columnDestinationUUID                = "destination_uuid"
 	columnDirectoryUUID                  = "directory_uuid"
 	columnEndTime                        = "end_time"
+	columnEntityName                     = "entity_name"
+	columnEntityType                     = "entity_type"
+	columnEntityUUID                     = "entity_uuid"
 	columnExtent                         = "extent"
 	columnExtentUUID                     = "extent_uuid"
 	columnHostAddr                       = "hostaddr"
 	columnHostName                       = "hostname"
+	columnInitiatorInfo                  = "initiator"
 	columnInputHostUUID                  = "input_host_uuid"
-	columnOriginZone                     = "origin_zone"
-	columnRemoteExtentPrimaryStore       = "remote_extent_primary_store"
+	columnIsMultiZone                    = "is_multi_zone"
+	columnKafkaCluster                   = "kafka_cluster"
```

Comment:
We will have a configuration for the location of the cluster.yaml file, and some generic file so that the integration test in travis-ci works.